.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/jcsda/spack
-  branch = jcsda_emc_spack_stack
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/jcsda/spack
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack
```

Comment:
I assume this is a temporary change for the sake of this PR?   Generally it's been not great for things to hinge an individual user owned accounts.

Reply:
Correct, but this is the only way for testing through the CI scripts. Once the spack PR is merged, the submodule pointer gets updated and `.gitmodules` reverted.